forge-gui/res/cardsfolder/upcoming/false_floor.txt

```diff
@@ -0,0 +1,8 @@
+Name:False Floor
+ManaCost:4
+Types:Artifact
+K:CARDNAME enters the battlefield tapped.
+R:Event$ Moved | ValidCard$ Creature | Destination$ Battlefield | ReplaceWith$ ETBTapped | ActiveZones$ Battlefield | Description$ Creatures enter the battlefield tapped.
+SVar:ETBTapped:DB$ ChangeZone | Origin$ All | Destination$ Battlefield | Tapped$ True | Defined$ ReplacedCard
+A:AB$ ChangeZoneAll | Cost$ 2 T Exile<1/CARDNAME> | ChangeType$ Creature.untapped | Origin$ Battlefield | Destination$ Exile | SorcerySpeed$ True | SpellDescription$ Exile all untapped creatures. Activate only as a sorcery.
```

Comment:
Is there some slick AI we should apply to this? Or, if it's just gross to make that work, should we do a `RemoveDeck`?

Reply:
Hmm, I don't remember if there's anything already e.g. in SpecialCardAi that's similar... It seems possible to write a special AI logic for this, but I prolly need a bit more time than now, so prolly a project for summer time... I guess RemoveDeck is OK for now if the AI is silly with it :)

Reply:
I'll try to remember to test it a bit... otherwise we can just let folks complain about it to alert us to the problem :)